simple_analog_clock.py

```diff
@@ -26,15 +26,15 @@ def draw_clock(h, m, s, pen): # Defining function, draw a clock, using pen turtl
     pen.up() # Dont want to draw a line
     pen.goto(0, 210) # Placement of pen tool
     pen.setheading(180) # turtle facing to the eft
-    pen.color("green") # Color
+    pen.color("white") # Color
     pen.pendown() # I want a line drawn at this point
-    pen.circle(210) # Draw circle 210 is radius of circle
+    pen.circle(310) # Draw circle 210 is radius of circle
 
 
     # Draw hands
     pen.penup() # Stop drawing line
     pen.goto(0,0) # Go to center
-    pen.setheading(90) # Face up
```

Comment:
Why is it supposed to be 90 and not 20?

Reply:
So since it is a circle we are dealing with, the computing that is required to form this circular clock works on degree setting angles. 90 degrees would equal straight up in this case.